assemblies/creating-code-server-workbenches.adoc

```diff
@@ -0,0 +1,16 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="creating-code-server-workbenches_{context}"]
+= Creating code-server workbenches
+
+[role='_abstract']
+You can create a blank workbench or import a workbench in code-server from several different sources.
```

Comment:
```suggestion
You can create a blank Jupyter Notebook or import a Jupyter Notebook in code-server from several different sources.
```

Reply:
not sure about this one, because the first section below really talks about starting (spawning, if we wanted to adopt this word, see the last segment of page URL in the attached screenshot) a workbench, but the second link talks about importing a Jupyter notebook

let me edit my edit suggestion to account for that...

![image](https://github.com/user-attachments/assets/3207397d-4f62-4c0c-860b-690802a69039)